src/drivers.rs

```diff
@@ -389,12 +389,7 @@ impl Driver<'_> {
                     .build();
                 let inspection = INSPECT_DRIVER.get_metadata(&inspect_opts)?;
 
-                let os_version = inspection.get_version().ok_or_else(|| {
-                    miette!(
-                        help = format!("Please check with the image author about using '{IMAGE_VERSION_LABEL}' to report the os version."),
-                        "Unable to get the OS version from the labels"
-                    )
-                })?;
+                let os_version = inspection.get_version().unwrap_or(0);
```

Comment:
Let's leave this function alone and instead call a different function for whatever the vanilla template needs. We want to try to keep the fedora templates and the vanilla templates separate as possible.

Reply:
I agree. I just did it like this because I couldn't figure out any easier way to fix this error. We should have some way to detect and declare what the base image is, and call different functions based on that, and remove the hard requirement for `os_version` in the tagging system, etc.